docs/news.md

```diff
@@ -1,5 +1,9 @@
 ---
 # Feel free to add content and custom Front Matter to this file.
 # To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults
-layout: page
+layout: page 
+title: In the News
+permalink: /news/
 ---
+
+{% include news/news.html %}
```

Comment:
I would have loved to write this as content in markdown, but unfortunately the minima jekyll theme is too restrictive to allow linking to an external site as a 'post'. So instead I ripped the HTML and edited it manually and just embed that here. It works quite well.